waspc/data/Generator/templates/server/src/actions/types.ts

```diff
@@ -0,0 +1,35 @@
+{{={= =}=}}
+{=! TODO: This template is exactly the same at the moment as one for action
+          types, consider whether it makes sense to address this in the future. =}
```

Comment:
Should this one say "same as query"? And similar for other.

Reply:
I can't believe I remembered to change them this time, but switched them up. Thanks!